README.md

```diff
@@ -126,7 +126,7 @@ and then register a URL type entry using the value of `CFBundleIdentifier` as th
 
 ### Callback URL(s)
 
-Callback URLs are the URLs that Auth0 invokes after the authentication process. Auth0 routes your application back to this URL and appends additional parameters to it, including a token. Since callback URLs can be manipulated, you will need to add your application's URL to your client's **Allowed Callback URLs for security**. This will enable Auth0 to recognize these URLs as valid. If omitted, authentication will not be successful.
+Callback URLs are the URLs that Auth0 invokes after the authentication process. Auth0 routes your application back to this URL and appends additional parameters to it, including a token. Since callback URLs can be manipulated, you will need to add your application's URL to your Application's **Allowed Callback URLs for security**. This will enable Auth0 to recognize these URLs as valid. If omitted, authentication will not be successful.
```

Comment:
> you will need to add your **application's** URL to your **Application's** Allowed Callback URLs 

Please try to make clearer that you refer to different things. I guess the first one is the RN app and the other is the Auth0 client